examples/advanced-routing/README.md

```diff
@@ -115,8 +127,10 @@ The `/tea` endpoint has routing rules configured for GET and POST requests. If y
 
 Send a PUT request and confirm the 404 Not Found response:
 
-```bash
+```shell
 curl --resolve cafe.example.com:$GW_PORT:$GW_IP http://cafe.example.com:$GW_PORT/tea -X PUT
+```
+```
```

Comment:
```suggestion
```html
```

Reply:
Does this make sense to do since it's console output and not an html page?

Reply:
maybe not 👍